styles.css

```diff
@@ -11,7 +11,7 @@ header {
     text-align: center;
 } 
 .container {
-    background-image: url('./img_1.jpg');
+    /* background-image: url('./images/img_1.jpg'); */
```

Comment:
remove commented out code across the whole app,
you already have it in the commit history, so you can always come back to it